docker_files/home/lisk/run.sh

```diff
@@ -1,4 +1,25 @@
 #!/bin/bash
 
 confd -backend env -onetime
-node app.js
+jq '
+def walk(f):
```

Comment:
What is this function for? Reduce the template to the used keys only?
Is it required?

Reply:
@abelboldu correct, this is the implementation of:
> remove all null values from the templated config.json (contains only explicitely set values)
it is a necessary workaround to make the template and the new configuration system coexist in 1.1 and will disappear in 1.2